### PR TITLE
Api validations completed

### PR DESCRIPTION
**Pull request by JacobR96**

### CODE REVIEW

File: app/models/celebrity.rb [hunk 1]
```diff
@@ -1,2 +1,15 @@
...
+      validates :name, length: { minimum: 2 }
+
+
+         validates :age, presence: true
+
+       validates :stared_in, presence: true
+       validates :stared_in, length: { minimum:10 }
+
+
+    validates :image, presence: true
```
Comment (elysemontano) [inline app/models/celebrity.rb]: Please clean up the indentation and spacing.

**Review by elysemontano — CHANGES_REQUESTED** [review]
Please clean up your indentation in your spec files and in validations.

File: app/models/celebrity.rb (hunk 1 above)
Comment (JacobR96) [reply app/models/celebrity.rb]: Fixed it

**Review by elysemontano — CHANGES_REQUESTED** [review]
Double check the indentation from your patch request on in your spec file.

**Review by elysemontano — APPROVED** [review]

**Pull request merged**